packages/manager/modules/sms/src/sms/components/service-status/service-status.html

```diff
@@ -0,0 +1,66 @@
+<div class="col-sm-4">
```

Comment:
**nitpick:** `col-sm`, `col-md-`, 'col-lg-` can only be directly child of HTML elements with `.row`
Ref: https://stackoverflow.com/questions/22053195/bootstrap-rows-and-columns-do-i-need-to-use-row/22053352#22053352

Reply:
It's fixed. I move this class.